lab_2/main.py

```diff
@@ -24,7 +51,14 @@ def create_zero_matrix(rows: int, columns: int) -> list:
     e.g. rows = 2, columns = 2
     --> [[0, 0], [0, 0]]
     """
-    pass
+    is_rows_columns_correct = ((not isinstance(rows, bool)) and (not isinstance(columns, bool)) and
+                               isinstance(rows, int) and isinstance(columns, int) and (rows > 0) and (columns > 0))
+    if not is_rows_columns_correct:
+        return []
+
+    zero_matrix = [[0] * columns for i in range(rows)]
```

Comment:
There is a convention to use `_` if value will not be used.

`[[0] * columns for _ in range(rows)]`